src/isIsogram.test.js

```diff
@@ -6,4 +6,25 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`true for an empty string`, () => {
+    expect(isIsogram('')).toBe(true);
+  });
+
+  it(`true for a single letter`, () => {
+    expect(isIsogram('a')).toBe(true);
+  });
+
+  it(`is case-insensitive`, () => {
+    expect(isIsogram('Ccart')).toBe(false);
```

Comment:
The test case for case insensitivity is correctly checking that 'Ccart' is not an isogram. Ensure that the `isIsogram` function implementation handles case insensitivity by converting all letters to the same case before checking for duplicates.